build-ci/platforms.rst

```diff
@@ -0,0 +1,46 @@
+###################
+Supported Platforms
+###################
+
+Introduction
+============
+
+The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centres to laptops (DMS-REQ-0308).
+The software will likely work on any Unix-like operating system but we limit our testing and support to a fixed set of operating systems.
+
+Patches may be accepted from developers who test on other operating systems.
+
+Definitions
+===========
+
+We use the following definitions when defining which operating systems are supported.
+
+* A *supported* operating system is one which is regularly tested by our continuous integration system and which we will ensure all tests pass.
+  The list of supported operating systems may change over time, an in particular, if we encounter difficulties where keeping an operating system running is taking excessive developer and CI resources, we reserve the right to drop it from the list.
+  Each supported operating system will be associated with a specific compiler that we use for testing.
+  Other compilers may work but are not tested.
+* The *baseline* operating system is the operating system we are expecting to use for official deployments.
+  It is the primary operating system at NCSA and the software *must* work on it.
+  This operating system will be used for integration and verification testing.
+
+Supported Operating Systems
+===========================
+
+The following operating systems are supported and tested by Jenkins.
+
+CentOS 7
+--------
+
+The baseline operating system is CentOS7 and all code *must* work on CentOS 7.
+We currently use the default CentOS 7 compiler.
+We aim to support the most recent versions of CentOS7 within 6 months of release, although we can not guarantee that all NCSA computers and Jenkins will be upgraded promptly.
+
+CentOS 6
+--------
+
+We currently support CentOS 6 with the devtoolset-3 compiler set.
+
+macOS
+-----
+
+We support the two most recent releases of the Apple macOS operating system using the Apple clang compilers.
```

Comment:
We don't test against the two most recent, do we?  And which XCodes do we actually test with?

Reply:
Apple doesn't give you a choice -- you get the XCode you get from the App Store based on the OS version. We test Sierra and many people run High Sierra. That seems to be good enough to say we support both.